linen_examples/imagenet/main.py

```diff
@@ -37,6 +38,10 @@
     'model_dir', default=None,
```

Comment:
Other examples use --workdir instead of --model_dir. Can we converge to --workdir everywhere?

Reply:
Done. Also updated https://github.com/google/flax/issues/231#issuecomment-716568710